lib/index.js

```diff
@@ -207,7 +207,9 @@ class GoodSlack extends Stream.Writable {
             payload: Stringify(payload)
         };
 
-        Wreck.post(this._config.url, data, callback);
+        Wreck.post(this._config.url, data)
+            .then((result) => callback(null, result))
+            .catch((error) => callback(error));
```

Comment:
Could be expressed as: `.catch(callback);`

Reply:
I included it this way for specificity and to match the format of the line above, but can change.